.github/workflows/podman-test.yaml

```
@@ -23,7 +23,7 @@ jobs:
 
     - name: Run Integration tests
       env:
-        PODMAN_EXEC_NODES: ${{ secrets.PODMAN_TEST_EXEC_NODES }}
+        PODMAN_EXEC_NODES: 10
```

Suggestion to add a comment.

```suggestion
        # This should ideally be a configuration in the GH repo (secret or variable).
        # This is currently hard-coded because GH won't expose secrets or variables to PRs created from forks.
        # Hopefully, variables (which are non-sensible by definition) will be passed to workflows triggered by PRs from forks.
        # See https://github.com/community/community/discussions/44322
        PODMAN_EXEC_NODES: 10
```